README.md

```diff
@@ -193,6 +193,21 @@ app.use(function *(){
 app.listen(3000);
 ```
 
+### Using koa@next
+
+```JS
+const errors = require('@google/cloud-errors')();
+const Koa = require('koa');
+const app = new Koa();
+
+// Will attach to 'error' event which includes
+// request/response lifecycle errors
+errors.koaNext(app);
```

Comment:
Quick question: I would assume that `koa@next` moves pretty fast. Do we think this integration would be stable, i.e. we won't be broken as `koa@next` changes?